src/weapons/Skin.java

```diff
@@ -3,97 +3,40 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import lombok.Getter;
+import lombok.Setter;
+import lombok.EqualsAndHashCode;
+
+@Getter
+@Setter
+@EqualsAndHashCode
 public class Skin {
 	private String name;
 	private List<String> weapons = new ArrayList<String>();
```

Comment:
Lombok generates a Setter method for weapons property which was not there before Lombok. Can you add the necessary annotations so that it only generates the Getter method?